testing/tests/search.test.js

```diff
@@ -14,6 +14,7 @@ describe("Site search", () => {
     // very confusing errors within the important tests themselves.
     await page.goto(testURL("/en-US/search-index.json"));
     // It's JSON but this asserts that page will be findable
+    // eslint-disable-next-line jest/no-standalone-expect
```

Comment:
This page is puppeteer-expect so perhaps we shouldn't have the eslint jest plugin even check this file.

Reply:
It's more complaining that there is an expect in the `beforeAll` function rather than a test block. Didn't want to remove it because I didn't know if it might have a side affect like being used for the page to be fully loaded